warwomb/utilities/seed.py

```diff
@@ -0,0 +1,120 @@
+import os
+import json
+import sqlalchemy as db
+from dotenv import main
+
+main.load_dotenv()
+
+db_engine = None
+db_metadata = None
+unit_factions_table = None
+unit_faction_links_table = None
+units_table = None
+
+faction_names_map = {
+    "AC": { "dbid": None, "name": "Aeternus Continuum" },
+    "EMP": { "dbid": None, "name": "Empyrean" },
+    "ISA": { "dbid": None, "name": "Iron Star Alliance" },
+    "MW": { "dbid": None, "name": "Marcher Worlds" }
+    }
+
+# TODO: add 'tbd' columns to unit data file
+unit_column_map = {
+    "name": "name",
+    "tbd": "speed",
+    "tbd": "strength",
+    "tbd": "melee_attack",
+    "tbd": "ranged_attack",
+    "tbd": "defense",
+    "tbd": "armor",
+    "tbd": "focus",
+    "tbd": "health",
+    "dc": "deployment_cost"
+}
+
+data_file = open('../data_files/warcaster_units.txt', 'r')
+unit_dictionary = json.loads(data_file.read())
+data_file.close()
+
+def create_db_engine():
+    global db_engine, db_metadata
+    db_engine = db.create_engine(os.getenv("DATABASE_URL"))
+    connection = db_engine.connect()
+    db_metadata = db.MetaData()
+    connection.close()
+
+def setup_tables():
```

Comment:
You also shouldn't need a `setup_tables` function, or even. You can set up tables already by using `python -m alembic upgrade head`